setup/fio-setup-basic

```diff
@@ -180,6 +203,32 @@ __numa_cpu_nodes()
 parse_numa_mem_policy
 parse_numa_cpu_nodes
 
+parse_cpus_allowed_policy()
+{
+        if [ "$cpus_allowed_policy" = "split" ]; then
+                cpus_allowed=$cpus_allowed
```

Comment:
thanks, what is the purpose of this self assignment? Especially it assigns cpus_allowed in parse_cpus_allowed_policy.

Reply:
yes, you are right.self assignment is wrong, have deleted.